jsv/features/homepage-lomita.feature

```diff
@@ -10,6 +10,7 @@ Feature: Lomita Homepage Layout
   #  Given the "stanford_jumpstart_home_lomita" homepage layout is not selected
     Given I am logged in as a user with the "administrator" role
     And I am on "admin/stanford-jumpstart/customize-design"
+    Then I press the "edit-layouts-stanford-jumpstart-home-panama-selector" button
```

Comment:
We need to change the page to something in case the home page was already set to the option we are testing for. The select button is disabled for the active home page option and is not clickable. This extra step remedies this.